test/support/test_config.h

```diff
@@ -117,7 +117,8 @@
 #endif
 #endif //!defined(_ENABLE_RANGES_TESTING)
 
-#define _ENABLE_STD_RANGES_TESTING _ONEDPL_CPP20_RANGES_PRESENT
+#define _ENABLE_STD_RANGES_TESTING ONEDPL_HAS_RANGES_API
+#define TEST_CPP20_SPAN_PRESENT (__cpp_lib_span >= 202002L)

```

Comment:
Could you replace the rest instances of `_ONEDPL_CPP20_SPAN_PRESENT` in the `std_ranges_test.h`?